### PR TITLE
Fix to the new pull request button

### PR DESCRIPTION
**Pull request by User00015**
Fixing an issue where the new pull request button was pointing to an incorrect repository. 

To reproduce:

1) Create a fresh new gitea
2) Register as a new user, admin or otherwise.
3) Create a new organization.
4) Go to the organizations dashboard, 
5) Create a repository under that organization and initilize it
6) Return to the organization dashboard and select the newly created repository
7) Click the green new pull request icon

Expected Result:
You are taken to a branch-to-branch merge with the repository

Actual result:
You are taken to a non-existent fork that assumes to exist that leads to a 404

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,4 @@ prime/
 *.snap
 *.snap-build
 *_source.tar.bz2
+/.vs
```
Comment (techknowlogick) [inline .gitignore]: Please revert the change to this file.

**Review by User00015 — COMMENTED** [review]
Removed.

**Comment by silverwind** [comment]
> Removed.

Diff still shows a newline added, you might want to remove it :)

**Review by User00015 — COMMENTED** [review]
Removed for real this time.

**Comment by Piraty** [comment]
please squash the commits and give it an appropriate comment

**Review by zeripath — CHANGES_REQUESTED** [review]
There's no need to squash commits - we squash them on merging.

**Pull request closed without merging**